### PR TITLE
Added spring cloud task support

### PR DESCRIPTION
**Pull request by marcingrzejszczak**
related issue https://github.com/spring-cloud/spring-cloud-sleuth/issues/1903

### CODE REVIEW

File: task/src/main/java/com/example/sleuthsamples/SpringCloudTaskApplication.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	}
+
+	@Bean
+	CommandLineRunner myCommandLineRunner(Tracer tracer) {
+		return args -> log.info("<ACCEPTANCE_TEST> <TRACE:{}> Hello from consumer", tracer.currentSpan().context().traceId());
+	}
+
+	@Bean
+	ApplicationRunner myApplicationRunner(Tracer tracer) {
+		return args -> log.info("<ACCEPTANCE_TEST> <TRACE:{}> Hello from producer", tracer.currentSpan().context().traceId());
```
Comment (cppwfs) [inline task/src/main/java/com/example/sleuthsamples/SpringCloudTaskApplication.java:33]: Probably want to rename producer to myApplicationRunner

File: task/src/main/java/com/example/sleuthsamples/SpringCloudTaskApplication.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+public class SpringCloudTaskApplication {
+	private static final Logger log = LoggerFactory.getLogger(SpringCloudTaskApplication.class);
+
+	public static void main(String... args) {
+		new SpringApplicationBuilder(SpringCloudTaskApplication.class).web(WebApplicationType.NONE).run(args);
+	}
+
+	@Bean
+	CommandLineRunner myCommandLineRunner(Tracer tracer) {
+		return args -> log.info("<ACCEPTANCE_TEST> <TRACE:{}> Hello from consumer", tracer.currentSpan().context().traceId());
```
Comment (cppwfs) [inline task/src/main/java/com/example/sleuthsamples/SpringCloudTaskApplication.java:28]: Probably want to rename consumer to myCommandLineRunner

File: task/README.adoc [hunk 3]
```diff
@@ -0,0 +1,5 @@
+= Spring Cloud Task
+
+We're wrapping `TaskExecutionListener`s and `CommandLineRunner` and `ApplicationRunner`.
```
Comment (cppwfs) [inline task/README.adoc:3]: Added a TaskExecutionListener to the sample to see if I could see the output in Zipkin.   I didn't see it appear.   But I probably messed something up.   Here is my additions: https://github.com/cppwfs/spring-cloud-sleuth-samples/commit/465a2ea07a50a7bd8257f2b81975d6b1702eb50c

File: task/src/main/java/com/example/sleuthsamples/SpringCloudTaskApplication.java (hunk 1 above)
Comment (marcingrzejszczak) [reply task/src/main/java/com/example/sleuthsamples/SpringCloudTaskApplication.java:33]: It's fixed for `producer` cause I require this text to be present in the `acceptance_tests` module :grimacing:

File: task/src/main/java/com/example/sleuthsamples/SpringCloudTaskApplication.java (hunk 2 above)
Comment (marcingrzejszczak) [reply task/src/main/java/com/example/sleuthsamples/SpringCloudTaskApplication.java:28]: It's fixed for `consumer` cause I require this text to be present in the `acceptance_tests` module :grimacing:

File: task/README.adoc (hunk 3 above)
Comment (marcingrzejszczak) [reply task/README.adoc:3]: I'll add the info to the `README`

**Comment by marcingrzejszczak** [comment]
done via https://github.com/spring-cloud-samples/spring-cloud-sleuth-samples/commit/1396cf73968062e00ae37c741b356b100f5617bb

**Pull request closed without merging**